usaspending_api/awards/v2/filters/filter_helpers.py

```diff
@@ -42,7 +47,51 @@ def date_or_fy_queryset(date_dict, table, fiscal_year_column, action_date_column
     return False, None
 
 
-def total_obligation_queryset(amount_obj, model):
+def sum_transaction_amount(qs, aggregated_name='transaction_amount', filter_types=award_type_mapping,
+                           calculate_totals=True):
+    """ Returns correct amount for transaction if loan (07, 08) vs all other award types (covers IDV)"""
```

Comment:
can you add in the docstring that this function returns a queryset? mainly so it's explicit for anyone using this function

Reply:
Added additional comments to clarify functions